### DISCUSSION TITLE
Low bitrates

### DISCUSSION THREAD

**Discussion by Alf1973** [aclap-dev/video-downloadhelper#943] [Help and Q&A]
### Your question

Why downloaded bitrates are video 250 kbps and audio 128?

### Addon & CoApp version and details

version: 9.0.2.12a1
target: mozilla
channel: beta
lang: fi
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"a7033f57307f4e0a"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:127.0) Gecko/20100101 Firefox/127.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive"
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_legacy_ui": false,
    "use_sidebar": false,
    "open_count_store": 2,
    "successfull_dl": 5
}

**Comment by GrampaWildWilly** [comment]
I pressed my forehead to my monitor & divined by clairvoyance what videos you are talking about.  But the smudge on my monitor is preventing me from revealing to you what I discovered.
